lib/grppc.gi

```diff
@@ -1491,18 +1491,7 @@ local G,           # common parent
     if eas=fail then
         home:=PcgsElementaryAbelianSeries([G,NT]);
         eas:=EANormalSeriesByPcgs(home);
-        cent:=function(pcgs,grpg,Npcgs,dep)
-              local i,j;
-                for i in grpg do
-                  for j in Npcgs do
-                    if DepthOfPcElement(pcgs,Comm(j,i))<dep then
-                      return false;
-                    fi;
-                  od;
-                od;
-                return true;
-              end;
-
+        cent:=PcClassFactorCentralityTest;
```

Comment:
Another great catch -- out of curiosity, how did you find this?

Reply:
The next version of gaplint warns about duplicate functions, I'm using that, there about 80 more duplicates in the library with at least 4 lines. Thanks for the feedback, I'll fix this up tomorrow @fingolfin